Source/santad/SNTDaemonControlController.m

```diff
@@ -214,14 +215,16 @@ - (void)setBundlesEnabled:(BOOL)bundlesEnabled reply:(void (^)())reply {
 #pragma mark GUI Ops
 
 - (void)setNotificationListener:(NSXPCListenerEndpoint *)listener {
-  SNTXPCConnection *c = [[SNTXPCConnection alloc] initClientWithListener:listener];
+  // This will leak the underlying NSXPCConnection when "fast user switching" occurs.
+  // It is not worth the trouble to fix. Maybe furtue self will feel differently.
```

Comment:
Spelling

Reply:
Also, is fixing this not just a case of invalidating any existing listener?

Reply:
That would work until it doesn't. Invalidating the existing listener would prompt that listener to reconnect. Invaliding the new listener. Prompting the new listener to reconnect. Invalidating the previous,  existing listener...You get the point.

This *should* be mitigated by [0], but it's racey [1] and not guaranteed to run before the new user becomes active.

This could also be triggered by manually killing the existing Santa process.

[0] https://github.com/google/santa/blob/master/Source/SantaGUI/SNTAppDelegate.m#L45-L47
[1] https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPMultipleUsers/Concepts/UserSwitchNotifications.html#Event%20Timing